frontend/src/layouts/BaseLayout.vue

```diff
@@ -76,10 +82,17 @@
             </connect-wallet>
           </div>
         </div>
-        <!-- Alpha warning -->
-        <div class="dark-toggle text-center text-negative text-bold q-my-md">
-          This is beta software. Use at your own risk.
-          <router-link class="hyperlink" :to="{ path: '/faq#security' }">Learn more</router-link>
+        <!-- Legacy warning -->
+        <div
+          v-if="!isAccountSetup && isAccountSetupLegacy"
+          class="dark-toggle text-center text-bold q-my-md q-pa-md"
+          style="border-radius: 15px"
+          :style="isDark ? 'color: #FFFAEB; background-color: #7c5e10' : 'color: #513C06; background-color: #fcefc7'"
+        >
+          We've upgraded our name resolution system. Use the
```

Comment:
This looks good, but I'm afraid it's easy to ignore.

<img width="1060" alt="Screen Shot 2021-10-05 at 1 48 16 PM" src="https://user-images.githubusercontent.com/1479802/136076108-cb398fc8-7d2e-4569-98c9-5e9fe5d88499.png">

I wonder if we should use a red background and text, and add some 🚨🚨🚨 to it for good measure 😅